docs/sphinx/source/stateoftheart_diagnostics/ensemble.rst

```diff
@@ -0,0 +1,149 @@
+- Plots the ensemble mean and standard deviation separately on two different maps.
+
+The `EnsembleZonal` class take zonal-averages `Lev-Lon` `xarray.Dataset` as input and performs the following functionalities:
+- Computes ensemble mean and standard deviation of the given input.
+
+The `PlotEnsembleZonal` class take zonal-averages `Lev-Lon` `xarray.Dataset` as input and performs the following functionalities:
+- Plots the ensemble mean and standard deviation of the computed statistics. 
+
+Structure
+---------
+
+* ``ensembleTimeseries.py``: contains the `EnsembleTimeseries` class.
+* ``plot_ensemble_timeseries.py``: contains the `PlotEnsembleTimeseries` class.
+* ``ensembleLatLon.py``: contains the `EnsembleLatLon` class.
+* ``plot_ensemble_latlon.py``: contains the `PlotEnsembleLatLon.py` class.
+* ``ensembleZonal.py``: contains the `EnsembleZonal` class.
+* ``plot_ensemble_zonal.py``: contains the `PlotEnsembleLatLon.py` class.
+* ``cli_timeseries_ensemble.py``: the command line interfance (CLI) script to run the ensemble-timeseries `1D` diagnostic.
+* ``cli_global_2D_ensemble.py``: the command line interfance (CLI) script to run the ensemble-2D-maps in `Lat-Lon` diagnostic.
+* ``cli_zonal_ensemble.py``: the command line interfance (CLI) script to run the ensemble-zonal `Lev-Lon` diagnostic.
+* ``util.py``: contains the `retrieve_merge_ensemble_data` and `compute_statistics` functions.
+* ``base.py``: contains the base class which contains functions for saving the output as png, pdf and netcdf.
+* ``config/diagnostics/ensemble/config_global_2D_ensemble.yaml``: config file for `cli_global_2D_ensemble.py`.
+* ``config/diagnostics/ensemble/config_timeseries_ensemble.yaml``: config file for `ensembleTimeseries.py`.
+* ``config/diagnostics/ensemble/config_zonalmean_ensemble.yaml``: config file for `ensembleZonal.py`.
+
+Input variables
+---------------
+
+In order to use the `Ensemble` module, a pre-processing step is required. To load and to merge the input data, `aqua.diagnostics.ensemble.util.retrieve_merge_ensemble_data` can be used which takes the list of paths of data or uses catalog entries. In this step one has to merge all the given `1D` timeseries, `2D` `Lat-Lon` Map and Zonal-averages `Lev-Lon` for `EnsembleTimeseries`, `EnsembleLatLon` and `EnsembleZonal` along a pesudo-dimension, respectively. The default dimension is simply named as `ensemble` and can be changed. One can load the data directly as `xarray.Dataset` or can use the `aqua` `Reader` class. For example loading and merging a 2D maps ensemble into an `xarray,Dataset`: 
+
+.. code-block:: python
+   
+   import glob
+   from  aqua.diagnostics.ensemble.util import retrieve_merge_ensemble_data
+   
+   file_list = glob.glob('/work/ab0995/a270260/pre_computed_aqua_analysis/*/historical-1990/atmglobalmean/netcdf/atmglobalmean.statistics_maps.2t.*_historical-1990.nc')
```

Comment:
I think we can keep as it is but in the medium term we would like to have an import from the Reader